Makefile

```diff
@@ -90,18 +101,24 @@ NIXPKGS_MANUAL_OUT = nixpkgs/manual
 all: $(NIXPKGS_MANUAL_OUT)
 
 $(NIXPKGS_MANUAL_OUT): $(NIXPKGS_MANUAL_IN) bootstrapify-docbook.sh bootstrapify-docbook.xsl layout.tt common.tt
-	./bootstrapify-docbook.sh $(NIXPKGS_MANUAL_IN)/share/doc/nixpkgs $(NIXPKGS_MANUAL_OUT) 'Nixpkgs manual' nixpkgs https://github.com/NixOS/nixpkgs/tree/master/doc
-	ln -sfn manual.html $(NIXPKGS_MANUAL_OUT)/index.html
+	@echo rm -rf $@
+	./bootstrapify-docbook.sh $(NIXPKGS_MANUAL_IN)/stable/share/doc/nixpkgs $(NIXPKGS_MANUAL_OUT)/stable 'Nixpkgs manual' nixpkgs https://github.com/NixOS/nixpkgs/tree/master/doc
+	ln -sfn manual.html $(NIXPKGS_MANUAL_OUT)/stable/index.html
+	# put unstable manual under $(NIXOS_MANUAL_OUT)/unstable
+	./bootstrapify-docbook.sh $(NIXPKGS_MANUAL_IN)/unstable/share/doc/nixpkgs $(NIXPKGS_MANUAL_OUT)/unstable 'Nixpkgs manual' nixpkgs https://github.com/NixOS/nixpkgs/tree/master/doc
+	ln -sfn manual.html $(NIXPKGS_MANUAL_OUT)/unstable/index.html
+	# alias for stable
+	ln -sfn stable/index.html $(NIXPKGS_MANUAL_OUT)/index.html
```

Comment:
This will orphan a `https://nixos.org/nixpkgs/manual/manual.html` page, that seems to be un-indexed, but still will exist.

“But there's an `rm`!” or so I thought initially, but it's a no-op print of `rm -rf`; `@echo` is an unintuitive construct in Makefiles. (And I'm not targeting you, this seems borrowed from another target.)

What about fixing the `rm -f`? I'm not sure it's a good idea, this will, periodically [(every twenty minutes or so)](https://github.com/NixOS/nixos-org-configurations/blob/517cabda83b1e0678e39e4dd8c221308e547d8ca/nixos-org/webserver.nix#L230) cause the manuals to briefly vanish from the server while they're being generated. Tried it locally and it's trivial to see it; this will end up confusing users, I think.

What should be done? Not sure. About the orphaned manual.html file? Maybe manually remove it. Should we `mv manual.html index.html` instead of symlinking to prevent unintended files to be present? Could we generate into a temp dir and as-atomically-as-possible pivot the manual to it instead of `rm` ing for the shortest amount of time the manuals?

Same comments about `rm` applies to the nixos manual, except there isn't a manual.html file there.